parsec/core/gui/tr/parsec_en.po

```diff
@@ -1758,7 +1758,7 @@ msgstr "Close"
 msgid "TEXT_NEW_VERSION_UP_TO_DATE"
 msgstr "Your PARSEC is up-to-date!"
 
-msgid "LABEL_NOTIFICATIONS"
+msgid "TEXT_NOTIFICATIONS_TITLE"
```

Comment:
Text is missing:
![image](https://user-images.githubusercontent.com/3187637/80189938-bf49b200-8613-11ea-87af-6aaa930dc188.png)